Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
-gem 'berkshelf', '~> 2.0'
-gem 'foodcritic', '~> 3.0'
-gem 'chefspec', '~> 3.0'
+gem 'berkshelf'
+gem 'foodcritic'
+gem 'rubocop'
+gem 'chefspec'
+gem 'test-kitchen'
+gem 'kitchen-vagrant'
```

Comment:
Can you use the Gemfile from https://github.com/opscode-cookbooks/apparmor/blob/master/Gemfile here.  It pins the versions and groups the dependencies so we can avoid installing Test Kitchen gems when we do Travis CI tests.